TEveEventDisplay/fcl/prolog.fcl

```diff
@@ -126,6 +130,7 @@ TEveEventDisplayCosmics : {
       addTrkExtTrajs		        : false
       addMCTraj				          : false 
    }
+   gdmlname :    "Offline/TEveEventDisplay/src/mu2e.gdml"
```

Comment:
This comment started out narrow: please refactor the code so that the default gdml file name appears in exactly one place and is propagated to the others by the use of @local::

But I see that the file has many configurations repeated wholesale with only minor changes.  So I am making a broader request.  Please refactor this so that there is a base configuration plus additional configurations that override  selected definitions.  This will be much easier to maintain.

Please also consider making a sub table to hold all of the "show" definitions.  The purpose is to reduce the number of items at the top level of configuration.